commandapi-core/src/main/java/dev/jorel/commandapi/Tooltip.java

```diff
@@ -90,7 +241,49 @@ public static <S> Tooltip<S>[] arrayOf(Tooltip<S>... tooltips) {
 	 * @return the mapping function from this tooltip into a StringTooltip
 	 */
 	public static <S> Function<Tooltip<S>, StringTooltip> build(Function<S, String> mapper) {
-		return t -> StringTooltip.of(mapper.apply(t.object), t.tooltip);
+		return t -> StringTooltip.ofMessage(mapper.apply(t.object), t.tooltip);
 	}
-	
+
+	/**
+	 * Converts an unformatted string to an unformatted tooltip by wrapping as with a {@link LiteralMessage}.
+	 *
+	 * If formatting is required, please see {@link #toMessage(BaseComponent...)},
+	 * or consider using the more modern adventure text api.
+	 *
+	 * @param string unformatted string tooltip
+	 * @return wrapped tooltip as a {@link LiteralMessage}
+	 */
+	public static Message toMessage(String string) {
+		return new LiteralMessage(string);
+	}
+
+	/**
+	 * Converts a formatted bungee text component to a native minecraft text component which can be used natively by brigadier.
+	 *
+	 * This supports all forms of formatting including entity selectors, scores,
+	 * click &amp; hover events, translations, keybinds and more.
+	 *
+	 * Note: the bungee component api is deprecated, and the adventure text component api should be used instead
+	 *
+	 * @param components array of bungee text components
+	 * @return native minecraft message object which can be used natively by brigadier.
+	 */
+	@Deprecated
```

Comment:
Deprecated method must have a corresponding `@deprecated` entry in the JavaDocs to point users to the alternative to using the deprecated method.

Reply:
Added in f500786